formbricks/formbricks
# Feat/highlight border

## What does this PR do?



Fixes #572  (issue)



https://github.com/formbricks/formbricks/assets/54475686/45e96b17-c4c0-4353-80bd-fe37fc2ef638

## Type of change




## How should this be tested?



- Go to the "look and feel" section of the settings and add / edit the highlight border
- Connect the demo app with a survey and the survey view div should have a border of the same highlight border color if ena bled.

## Checklist
